packages/stream/solana/types.ts

```diff
@@ -229,6 +259,99 @@ export interface DecodedStream {
   fundsUnlockedAtLastRateChange: BN;
 }
 
+export enum SolanaProxyContractErrorCode {
```

Comment:
Is this enum auto-generated?

If not, then it could easily become out-of-sync. Anchor has `translateError` and `parseIdlErrors` apis for IDL errors. Can we use it?